### PR TITLE
Show proper data in runtimes list and details views

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- Show proper data in runtimes list and details views

**Related issue(s)**

### CODE REVIEW

File: compass/src/components/Shared/StatusBadge/StatusBadge.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import { Badge } from 'fundamental-react/lib/Badge';
+
+class StatusBadge extends React.Component {
+  PropTypes = {
+    status: PropTypes.string.isRequired,
+  };
+
+  processStatus() {
+    const { status } = this.props;
+    let type = 'warning';
```
Comment (Wawrzyn321) [inline compass/src/components/Shared/StatusBadge/StatusBadge.js]:
```suggestion
    let type;
```
This will do alone, as it's gonna be overridden below.

**Comment by Wawrzyn321** [comment]
Runtime header data isn't visible on this screen size.
image

**Comment by Wawrzyn321** [comment]
Also, please change the presentation components from class to function ones, and please add PropTypes validation.

File: compass/src/components/Runtimes/Runtimes.component.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+import LuigiClient from '@kyma-project/luigi-client';
+
+import CreateRuntimeForm from './CreateRuntimeForm/CreateRuntimeForm';
+import StatusBadge from '../Shared/StatusBadge/StatusBadge';
+import GenericList from '../../shared/components/GenericList/GenericList';
+import { EMPTY_TEXT_PLACEHOLDER } from '../../shared/constants'
+
+class Runtimes extends React.Component {
+  static propTypes = {
+    runtimes: PropTypes.object.isRequired,
```
Comment (Wawrzyn321) [inline compass/src/components/Runtimes/Runtimes.component.js]: We can add _addRuntime_ and _deleteRuntime_ here.

File: compass/src/components/Runtimes/Runtimes.component.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+          .catch(() => {});
+      },
+    },
+  ];
+
+  render() {
+    const runtimesQuery = this.props.runtimes;
+
+    const runtimes =
+      (runtimesQuery &&
```
Comment (Wawrzyn321) [inline compass/src/components/Runtimes/Runtimes.component.js]: Do we need the check for _ runtimesQuery_ here and below? PropTypes will already raise clear error when _this.props.runtimes_ is not supplied.

File: compass/src/components/Shared/ResourceNotFound.component.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    <Panel className="fd-has-margin-large">
+      <PanelBody className="fd-has-text-align-center fd-has-type-4">
+        Such {resource} doesn't exists for this Tenant.
+      </PanelBody>
+    </Panel>
+  </>
+);
+
+const navigateToList = breadcrumb => {
+  const path = breadcrumb.toLowerCase();
```
Comment (Wawrzyn321) [inline compass/src/components/Shared/ResourceNotFound.component.js]: Please check if we really need it here.

File: compass/src/shared/constants.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+const EMPTY_TEXT_PLACEHOLDER = '-'
+
+module.export = {
+  EMPTY_TEXT_PLACEHOLDER
+}
```
Comment (Wawrzyn321) [inline compass/src/shared/constants.js]: Why use _module.exports_? ES6 _export const_ would do, as we use it in most other places.

File: compass/src/components/Runtimes/Runtimes.component.js (hunk 2 above)
Comment (dariadomagala) [reply compass/src/components/Runtimes/Runtimes.component.js]: We don't have `addRuntime` at this point, but I will add `deleteRuntime`, good point :)

**Pull request merged**